addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -10,20 +10,22 @@ var QWeb = core.qweb;
 
 models.PosModel = models.PosModel.extend({
     create_printer: function (config) {
-        var url = config.proxy_ip || '';
-        if(url.indexOf('//') < 0) {
-            url = window.location.protocol + '//' + url;
-        }
-        if(url.indexOf(':', url.indexOf('//') + 2) < 0 && window.location.protocol !== 'https:') {
-            url = url + ':8069';
+        if (!config.device_identifier && config.printer_type == 'iot'){
+            var url = config.proxy_ip || '';
+            if(url.indexOf('//') < 0) {
+                url = window.location.protocol + '//' + url;
+            }
+            if(url.indexOf(':', url.indexOf('//') + 2) < 0 && window.location.protocol !== 'https:') {
+                url = url + ':8069';
+            }
+            return new Printer(url, this);
         }
-        return new Printer(url, this);
     },
 });
 
 models.load_models({
     model: 'restaurant.printer',
-    fields: ['name','proxy_ip','product_categories_ids'],
+    fields: ['name','proxy_ip','product_categories_ids', 'printer_type'],
     domain: null,
     loaded: function(self,printers){
         var active_printers = {};
```

Comment:
@aprieels Do you think we should check here that the option multi-print is installed?

Reply:
@aprieels And shouldn't we put a domain looking like [('id', 'in', self.config.printer_ids)]?

Reply:
Yes adding a domain would be better! But I'm not sure the check is needed, we already have this in the `pos.config` model:
```
    @api.onchange('is_order_printer')
    def _onchange_is_order_printer(self):
        if not self.is_order_printer:
            self.printer_ids = [(5, 0, 0)]
```

Reply:
ok, I'll just remove the useless 'if' statement, and we'll see later if we want to modify the demo data, and to put a domain as there are multiple models that could have a better domain.